src/libraries/Common/src/Interop/Windows/Kernel32/Interop.HandleInformation.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Kernel32
+    {
+        [Flags]
+        internal enum HandleFlags: uint
+        {
+            None = 0,
+            Inherit = 1,
+            ProtectFromClose = 2
```

Comment:
We generally try to use the same naming as in the native signatures, e.g.
HANDLE_FLAG_INHERIT
HANDLE_FLAG_PROTECT_FROM_CLOSE
etc.

Reply:
Looks like half of the WSA interop code does not follow the guideline. I've used this naming for consistency with those parts (eg. `AsyncEventBits`). Maybe worth to fix those.

Reply:
Ok.  Let's use the desired naming, and separately fix the interop code that's not following it.